Anton_Usertiger/02.GIT.Local.md

```diff
@@ -0,0 +1,187 @@
+# GIT.02.Local
+# Commands, @USERTIGER:
+```bash
+  # Init
+  396  date >> 1.txt
+  397  git add 1.txt
+  398  git commit -m "1.txt included"
+  399  date >> 2.txt
+  400  git add 2.txt
+  401  git commit -m "2.txt included"
+  402  git status
+  403  git log
+  # Next step - Create branch dev
+  404  git checkout -b dev
+  405  date >> 3.txt
+  406  git add 3.txt
+  407  git commit -m "3.txt added"
+  408  date >> 4.txt
+  409  git add 4.txt
+  410  git commit -m "4.txt added"
+  411  git status
+  412  git log
+  # Next step - Create branch features/do_one
+  413  git checkout -b features/do_one
+  414  date >> 5.txt
+  415  git add 5.txt
+  416  git commit -m "5txt"
+  417  git status
+  418  git log
+  # Next step - Create branch hotfix/we_gonna_die
+  419  git checkout master
+  420  git checkout -b hotfix/we_gonna_die
+  421  date >> hotfix.txt
+  422  git add hotfix.txt
+  423  git commit -m "hotfix"
+  424  git status
+  425  git log
+  # Next step - Merge to master
+  426  git checkout master
+  427  git merge dev
+  428  git merge features/do_one
+  429  git checkout hotfix/we_gonna_die
+  430  git log --oneline
+  431  git checkout master
+  432  git cherry-pick 0af8b31
+  # Next step - Merge hotfix to dev & features/do_one
+  433  git checkout dev
+  434  git cherry-pick 0af8b31
+  435  git checkout features/do_one
+  436  git cherry-pick 0af8b31
+  # Next step - Finish
+  437  git checkout master
+  438  git show
+  439  history
+```
+#Commit Master:
+
+commit 4a9f28eec1254f94ac3e9c121695c3029e3f9546 (HEAD -> master)
```

Comment:
Please use git markdown to display lists of commits, like you used for output of history command

Reply:
Thanks, solved